<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SKDrawTool</class>
 <widget class="QDialog" name="SKDrawTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图形修改</string>
  </property>
  <widget class="QGraphicsView" name="MainView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>621</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_table">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>161</width>
     <height>431</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>背景色</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>样式</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>前景色</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>线宽度</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>线颜色</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>线样式</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>透明度</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>层次</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>文本内容</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>显示属性</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="BackColor">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="m_pattern">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_linewidth">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="m_linestyle">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/linetype/images/LineType/L0.ico</normaloff>:/linetype/images/LineType/L0.ico</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/linetype/images/LineType/L1.ico</normaloff>:/linetype/images/LineType/L1.ico</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/linetype/images/LineType/L2.ico</normaloff>:/linetype/images/LineType/L2.ico</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/linetype/images/LineType/L3.ico</normaloff>:/linetype/images/LineType/L3.ico</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/linetype/images/LineType/L4.ico</normaloff>:/linetype/images/LineType/L4.ico</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/linetype/images/LineType/L5.ico</normaloff>:/linetype/images/LineType/L5.ico</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/linetype/images/LineType/L6.ico</normaloff>:/linetype/images/LineType/L6.ico</iconset>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="m_alpha">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QPushButton" name="FrontColor">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="LineColor">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_SaveItemBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_Cancel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <action name="action_Group">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/group.ico</normaloff>:/standardtool/images/standardtool/group.ico</iconset>
   </property>
   <property name="text">
    <string>组合</string>
   </property>
   <property name="toolTip">
    <string>组合</string>
   </property>
  </action>
  <action name="action_Disgroup">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/ungroup.ico</normaloff>:/standardtool/images/standardtool/ungroup.ico</iconset>
   </property>
   <property name="text">
    <string>拆分</string>
   </property>
   <property name="toolTip">
    <string>拆分</string>
   </property>
  </action>
  <action name="action_LeftAlign">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/leftalign.ico</normaloff>:/standardtool/images/standardtool/leftalign.ico</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
   <property name="toolTip">
    <string>左对齐</string>
   </property>
  </action>
  <action name="action_RightAlign">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/rightalign.ico</normaloff>:/standardtool/images/standardtool/rightalign.ico</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
   <property name="toolTip">
    <string>右对齐</string>
   </property>
  </action>
  <action name="action_VCenter">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/vercenalign.ico</normaloff>:/standardtool/images/standardtool/vercenalign.ico</iconset>
   </property>
   <property name="text">
    <string>垂直居中</string>
   </property>
   <property name="toolTip">
    <string>垂直居中</string>
   </property>
  </action>
  <action name="action_TopAlign">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/topalign.ico</normaloff>:/standardtool/images/standardtool/topalign.ico</iconset>
   </property>
   <property name="text">
    <string>上对齐</string>
   </property>
   <property name="toolTip">
    <string>上对齐</string>
   </property>
  </action>
  <action name="action_ButtomAlign">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/bottomalign.ico</normaloff>:/standardtool/images/standardtool/bottomalign.ico</iconset>
   </property>
   <property name="text">
    <string>下对齐</string>
   </property>
   <property name="toolTip">
    <string>下对齐</string>
   </property>
  </action>
  <action name="action_HCenter">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/horcenalign.ico</normaloff>:/standardtool/images/standardtool/horcenalign.ico</iconset>
   </property>
   <property name="text">
    <string>水平居中</string>
   </property>
   <property name="toolTip">
    <string>水平居中</string>
   </property>
  </action>
  <action name="action_SameWidth">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/makesamewidth.ico</normaloff>:/standardtool/images/standardtool/makesamewidth.ico</iconset>
   </property>
   <property name="text">
    <string>等宽度</string>
   </property>
   <property name="toolTip">
    <string>等宽度</string>
   </property>
  </action>
  <action name="action_SameHeight">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/makesameheight.ico</normaloff>:/standardtool/images/standardtool/makesameheight.ico</iconset>
   </property>
   <property name="text">
    <string>等高度</string>
   </property>
   <property name="toolTip">
    <string>等高度</string>
   </property>
  </action>
  <action name="action_SameWidthHeight">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/makesamewihi.ico</normaloff>:/standardtool/images/standardtool/makesamewihi.ico</iconset>
   </property>
   <property name="text">
    <string>等宽等高</string>
   </property>
   <property name="toolTip">
    <string>等宽等高</string>
   </property>
  </action>
  <action name="action_VMirror">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/vermirror.ico</normaloff>:/standardtool/images/standardtool/vermirror.ico</iconset>
   </property>
   <property name="text">
    <string>垂直镜像</string>
   </property>
   <property name="toolTip">
    <string>垂直镜像</string>
   </property>
  </action>
  <action name="action_HMirror">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/hormirror.ico</normaloff>:/standardtool/images/standardtool/hormirror.ico</iconset>
   </property>
   <property name="text">
    <string>水平镜像</string>
   </property>
   <property name="toolTip">
    <string>水平镜像</string>
   </property>
  </action>
  <action name="action_DrawLine">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/drawtool/images/drawtool/line.ico</normaloff>:/drawtool/images/drawtool/line.ico</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
   <property name="toolTip">
    <string>绘制直线</string>
   </property>
  </action>
  <action name="action_DrawFoldLine">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/drawtool/images/drawtool/foldline.ico</normaloff>:/drawtool/images/drawtool/foldline.ico</iconset>
   </property>
   <property name="text">
    <string>折线</string>
   </property>
   <property name="toolTip">
    <string>绘制折线</string>
   </property>
  </action>
  <action name="action_DrawCircleLine">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/drawtool/images/drawtool/quxing.ico</normaloff>:/drawtool/images/drawtool/quxing.ico</iconset>
   </property>
   <property name="text">
    <string>曲线/圆弧</string>
   </property>
   <property name="toolTip">
    <string>绘制曲线/圆弧</string>
   </property>
  </action>
  <action name="action_DrawFreeLine">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/drawtool/images/drawtool/freeline.ico</normaloff>:/drawtool/images/drawtool/freeline.ico</iconset>
   </property>
   <property name="text">
    <string>自由直线</string>
   </property>
   <property name="toolTip">
    <string>绘制自由直线</string>
   </property>
  </action>
  <action name="action_DrawRect">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/rect.ico</normaloff>:/standardtool/images/standardtool/rect.ico</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="toolTip">
    <string>绘制矩形</string>
   </property>
  </action>
  <action name="action_DrawRoundRect">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/drawtool/images/drawtool/roundrect.ico</normaloff>:/drawtool/images/drawtool/roundrect.ico</iconset>
   </property>
   <property name="text">
    <string>圆角矩形</string>
   </property>
   <property name="toolTip">
    <string>绘制圆角矩形</string>
   </property>
  </action>
  <action name="action_Circle">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/drawtool/images/drawtool/ellpse.ico</normaloff>:/drawtool/images/drawtool/ellpse.ico</iconset>
   </property>
   <property name="text">
    <string>椭圆/圆</string>
   </property>
   <property name="toolTip">
    <string>绘制椭圆/圆</string>
   </property>
  </action>
  <action name="action_PoloyRect">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/polygon.ico</normaloff>:/standardtool/images/standardtool/polygon.ico</iconset>
   </property>
   <property name="text">
    <string>多边形</string>
   </property>
   <property name="toolTip">
    <string>绘制多边形</string>
   </property>
  </action>
  <action name="action_Arc">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/drawtool/images/drawtool/Arcitem.ico</normaloff>:/drawtool/images/drawtool/Arcitem.ico</iconset>
   </property>
   <property name="text">
    <string>扇形</string>
   </property>
   <property name="toolTip">
    <string>绘制扇形</string>
   </property>
  </action>
  <action name="action_DeleteItem">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/delete.ico</normaloff>:/standardtool/images/standardtool/delete.ico</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除图形</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_DrawText">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/drawtool/images/drawtool/text.ico</normaloff>:/drawtool/images/drawtool/text.ico</iconset>
   </property>
   <property name="text">
    <string>文本</string>
   </property>
   <property name="toolTip">
    <string>文本</string>
   </property>
  </action>
  <action name="action_movetoplest">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/bringtotop.ico</normaloff>:/standardtool/images/standardtool/bringtotop.ico</iconset>
   </property>
   <property name="text">
    <string>移到最上层</string>
   </property>
   <property name="toolTip">
    <string>移到最上层</string>
   </property>
  </action>
  <action name="action_movedownlest">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/sendtobottom.ico</normaloff>:/standardtool/images/standardtool/sendtobottom.ico</iconset>
   </property>
   <property name="text">
    <string>移到最下层</string>
   </property>
   <property name="toolTip">
    <string>移到最下层</string>
   </property>
  </action>
  <action name="action_movefore">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/bringforward.ico</normaloff>:/standardtool/images/standardtool/bringforward.ico</iconset>
   </property>
   <property name="text">
    <string>上移一层</string>
   </property>
   <property name="toolTip">
    <string>上移一层</string>
   </property>
  </action>
  <action name="action_moveback">
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/standardtool/images/standardtool/bringbackward.ico</normaloff>:/standardtool/images/standardtool/bringbackward.ico</iconset>
   </property>
   <property name="text">
    <string>下移一层</string>
   </property>
   <property name="toolTip">
    <string>下移一层</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
